packages/block/src/block.ts

```diff
@@ -564,13 +564,25 @@ export class Block {
     }
   }
 
+  private calcExcessDataGas(parentHeader: BlockHeader, newBlobs: number) {
```

Comment:
I think this duplicates [this](https://github.com/ethereumjs/ethereumjs-monorepo/blob/e4f5a42cd403d624029b0d8cac58a832a3219c05/packages/block/src/helpers.ts#L91) and I think this makes more sense as a standalone function than as part of the block class since there's nothing intrinsic to the block class required for this function to work.

Reply:
You are right, but that method is not called anywhere from inside the block package (which caused failing tests). It does not handle the edge case that parent block does not have 4844 activated and the block itself is the first 4844 block, will fix.

Reply:
Have removed this method and instead used the helper one